libs/ui/src/lib/drag-and-drop-file-input/drag-and-drop-file-input.component.css

```diff
@@ -0,0 +1,11 @@
+ngx-dropzone {
+  height: auto;
+  align-items: flex-start;
+  color: black;
+  border: none;
+  border-radius: 0;
+}
+
+ngx-dropzone-label {
```

Comment:
We use tailwindcss for the styles, it is made of many helper classes to set the `display`, `margin`, `padding` etc..
You almost never need to set css for this.

https://tailwindcss.com/docs/margin

It's the same for the color, border etc.. get inspired from https://github.com/geonetwork/geonetwork-ui/blob/master/libs/ui/src/lib/record-preview-list/record-preview-list.component.html for example.